msgpack-core/src/main/java/org/msgpack/core/MessageUnpacker.java

```diff
@@ -125,20 +132,43 @@
      */
     public MessageUnpacker(MessageBufferInput in)
     {
-        this(in, MessagePack.DEFAULT_CONFIG);
+        this.in = checkNotNull(in, "MessageBufferInput is null");
     }
 
-    /**
-     * Create an MessageUnpacker
-     *
-     * @param in
-     * @param config configuration
-     */
-    public MessageUnpacker(MessageBufferInput in, MessagePack.Config config)
+    public MessageUnpacker setAllowStringAsBinary(boolean enabled)
```

Comment:
Do we need these configuration setters in MessageUnpacker? I think these method should be moved to the factory (or config) side.